firefox-ios/firefox-ios-tests/Tests/XCUITests/BaseTestCase.swift

```diff
@@ -428,6 +427,24 @@ class BaseTestCase: XCTestCase {
         navigator.nowAt(SettingsScreen)
         app.buttons["Done"].waitAndTap()
     }
+    func waitForElementsToExist(_ elements: [XCUIElement], timeout: TimeInterval = TIMEOUT, message: String? = nil) {
+        var elementsDict = [XCUIElement: String]()
+        for element in elements {
+            elementsDict[element] = "exists == true"
+        }
+        let expectations = elementsDict.map(
+            {
```

Comment:
The lint error is still in this line

Reply:
@isabelrios Hoping this should fix the lint error. I have `0.57.0` installed through `homebrew` locally
<img width="508" alt="Screenshot 2024-11-04 at 1 09 57 PM" src="https://github.com/user-attachments/assets/3bb71a3c-e2f4-4876-b633-fcd804f4d955">